### PR TITLE
`notification_hub_namespace` waitLRO

### PR DESCRIPTION
**Pull request by shu-ying789**
linked to https://github.com/hashicorp/terraform-provider-azurerm/issues/15641

test result:
notificate2

### CODE REVIEW

File: internal/services/notificationhub/notification_hub_namespace_resource_test.go [hunk 1]
```diff
@@ -40,15 +40,15 @@ func TestAccNotificationHubNamespace_updateTag(t *testing.T) {
 				check.That(data.ResourceName).Key("tags.%").HasValue("1"),
 			),
 		},
-		data.ImportStep(),
+		data.ImportStep("namespace_type"),
```
Comment (stephybun) [inline internal/services/notificationhub/notification_hub_namespace_resource_test.go:43]: Why do we need to add `namespace_type` here? Is it not returned by the API or is it not set in the read function?

**Review by stephybun — CHANGES_REQUESTED** [review]

File: internal/services/notificationhub/notification_hub_namespace_resource_test.go (hunk 1 above)
Comment (ms-henglu) [reply internal/services/notificationhub/notification_hub_namespace_resource_test.go:43]: It's not returned, and refs https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/notificationhub/notification_hub_namespace_resource_test.go#L28

Comment (katbyte) [reply internal/services/notificationhub/notification_hub_namespace_resource_test.go:43]: other tests also do it

**Review by katbyte — APPROVED** [review]
Thanks @shu-ying789 - LGTM 🌴

**Pull request merged**